Assets/css/styles.css

```diff
@@ -0,0 +1,351 @@
+:root {
+  --main-white: #f0f0f0;
+  --main-red: #be3144;
+  --main-blue: #45567d;
+  --main-gray: #303841;
+  font-size: 18px;
+  margin: 0;
+  padding: 0%;
+}
+
+/* Base reset */
+* {
+  margin: 0;
+  padding: 0;
+}
+
+/* box-sizing and font sizing */
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
+}
+
+html {
+  box-sizing: border-box;
+  font-size: 62.5%;
+  scroll-behavior: smooth;
+}
+
+@media (max-width: 75em) {
```

Comment:
Media queries should be at the bottom of the CSS file and combine all selectors that are being changed for that screensize.

Reply:
changed